website/community/syncs.md

```diff
@@ -32,6 +36,8 @@ Here are some upcoming calls for convenience.
 
 ![Upcoming calls](/assets/images/upcoming-community-calls.png)
 
+
```

Comment:
are these intentional ?

Reply:
not intentional. side effect of copy pasting links. fixed!